modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,142 @@
+= Use Flow Designer to Configure a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Flow Designer is a tool in Anypoint Design Center that enables you to create apps visually. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To use Flow Designer to configure a connector:
```

Comment:
These two steps don't configure a connector. They are steps for creating an app that consists of two connectors.

Moreover, "To use Flow Designer to configure a connector" contradicts your intro sentence, which says that Flow Designer "enables you to create apps visually". Which is it? I can use FD to "create apps visually" or to "configure a connector"?

Also, the intro sentence doesn't introduce the concept of connectors in your topic. In fact, you first use the term "connector" without explaining how the concept is relevant to Flow Designer.